src/pages/en/eventregistration/index.html

```diff
@@ -0,0 +1,8 @@
+---
+lang_en: true
+page-title: Open-Source Project Corona-Warn-App - Event QR Code
+page-description: 
```

Comment:
@svengabr I've missed to add the page description in English. 

For the German description I used the text above the "create qr code" button on the start page in the android app.

Reply:
I have added the english texts, thanks!